data-prepper-plugins/otel-trace-group-prepper/src/test/java/com/amazon/dataprepper/plugins/prepper/oteltracegroup/OTelTraceGroupPrepperConfigTests.java

```diff
@@ -0,0 +1,33 @@
+package com.amazon.dataprepper.plugins.prepper.oteltracegroup;
+
+import com.amazon.dataprepper.model.configuration.PluginSetting;
+import com.amazon.dataprepper.plugins.sink.elasticsearch.ConnectionConfiguration;
```

Comment:
We can revisit the pros/cons of cross-plugin dependencies later, this works for now.